spec/unit/util/purgeable_spec.rb

```diff
@@ -15,8 +15,10 @@
       'spec/fixtures/unit/util/purgeable/managed_one/expected_1',
       'spec/fixtures/unit/util/purgeable/managed_one/new_1',
       'spec/fixtures/unit/util/purgeable/managed_one/managed_subdir_1',
+      'spec/fixtures/unit/util/purgeable/managed_one/managed_symlink_dir',
       'spec/fixtures/unit/util/purgeable/managed_one/managed_subdir_1/subdir_expected_1',
       'spec/fixtures/unit/util/purgeable/managed_one/managed_subdir_1/subdir_new_1',
+      'spec/fixtures/unit/util/purgeable/managed_one/managed_subdir_1/managed_symlink_file',
```

Comment:
Only "managed" symlinks appear in this list of desired contents, which is what makes them "managed".